pages.zh/common/rsync.md

```diff
@@ -0,0 +1,36 @@
+# rsync
+
+> 一种快速，通用，远程（和本地）文件复制工具
```

Comment:
Missing period?
```suggestion
> 一种快速，通用，远程（和本地）文件复制工具。
```

Reply:
@xiaolong-666 It's actually supposed to be a description, and it _can_ be long and have multiple sentences.  It's not supposed to be a subtitle, so a period should be at the end.

Reply:
Take a look at some other pages for reference, such a [this](https://github.com/tldr-pages/tldr/blob/master/pages.zh/common/adb-shell.md).  (it actually uses an English period, not a Chinese period, but I'm working on a big PR to fix those discrepancies across all of the Chinese pages :wink:)

Reply:
Thanks @bl-ue , the next PR will use this format